xml/System.Diagnostics.Tracing/EventCounter.xml

```diff
@@ -109,8 +109,15 @@ For an example of how to use the <xref:System.Diagnostics.Tracing.EventCounter>
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Removes the counter from set that the EventSource will report on.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+    ## Remarks
+
+    After being disposed, this counter will do nothing and its resource will be reclaimed if all references to it are removed. If an EventCounter is not explicitly disposed it will be cleaned up automatically when the EventSource it is attached to dies.
```

Comment:
```suggestion
After being disposed, this counter will do nothing and its resource will be reclaimed if all references to it are removed. If an EventCounter is not explicitly disposed it will be cleaned up automatically when the <xref:<xref:System.Diagnostics.Tracing.EventSource> it is attached to dies.

```